hack/yaml-lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then 
+  yamllint --config-data "{extends: default, rules: {line-length: {level: warning, max: 120}}}" ./examples/ ./installer/
```

Comment:
This would change the logic from what was in Travis, but we may want to lint `modules` as well.  I'm not sure why that wasn't added when the Travis logic landed in coreos/tectonic-installer#3268.

Reply:
Added `modules`

Reply:
> Added `modules`

Following up on this, @rabugopsl tried and [turned up some errors][1].  So we're punting on including `modules` for now, and we'll revisit after this PR lands.

[1]: https://travis-ci.org/openshift/installer/jobs/411456538#L430